testsetups/ocr.go

```diff
@@ -230,6 +238,73 @@ func (t *OCRSoakTest) checkLatestRound(expectedValue, roundNumber int) {
 	}
 }
 
+// checkCurrentResponseFromAnswerUpdatedEvent subscribes to the event log for AnswerUpdated event and
+// verifies if the answer is matching with the expected value
+func (t *OCRSoakTest) subscribeToAnswerUpdatedEvent(ctx context.Context) {
+	contractABI, err := ethereum.OffchainAggregatorMetaData.GetAbi()
+	Expect(err).ShouldNot(HaveOccurred(), "Getting contract abi for OCR shouldn't fail")
+	query := goeath.FilterQuery{
+		Addresses: []common.Address{},
+	}
+	for i := 0; i < len(t.ocrInstances); i++ {
+		query.Addresses = append(query.Addresses, common.HexToAddress(t.ocrInstances[i].Address()))
+	}
+	eventLogs := make(chan types.Log)
+	sub, err := t.chainClient.SubscribeFilterLogs(context.Background(), query, eventLogs)
+	if err != nil {
+		Expect(err).ShouldNot(HaveOccurred(), "Subscribing to contract event log in OCR instance shouldn't fail")
+	}
+	ocr := t.ocrInstances[0]
+	go func() {
+		//defer GinkgoRecover()
+
+		for {
+			select {
+			case err := <-sub.Err():
+				Expect(err).ShouldNot(HaveOccurred(), "Retrieving event subscription log in OCR instances shouldn't fail")
+				continue
+			case vLog := <-eventLogs:
+				// the first topic is the hashed event signature
+				eventDetails, err := contractABI.EventByID(vLog.Topics[0])
+				Expect(err).ShouldNot(HaveOccurred(), "Getting event details for OCR instances shouldn't fail")
+				// The following is a generic details of any event received.
+				/*parsedEvent, err := contractABI.Unpack(eventDetails.Name, vLog.Data)
```

Comment:
Do we still need this or is it something that is just useful to keep commented?

Reply:
Actually looking for suggestions here. There are multiple events published. If we think we might need to capture all such event details, I can remove the comments. Otherwise I can get rid of this snippet.
This actually corresponds to this part of pr description:

> 
Additionally the other event details can also be captured in logs ( commented code in ocr.go), if it's needed.

Reply:
I guess I am not sure what all is in the event off the top of my head. It might be worthwhile keeping the comment in for anyone that wants to tap into it for a quick test in the future.